pkg/js/README.md

```diff
@@ -90,6 +90,8 @@ class Options {
 }
 ```
 
+NB: This _anonymous_ class must have an unnamed _factory constructor_.
```

Comment:
_NB_ means [_Nota Bene_](https://en.wikipedia.org/wiki/Nota_bene). I can remove it if it is confusing.